invenio_app_rdm/records_ui/views/records.py

```diff
@@ -75,6 +76,43 @@ def open(self):
 #
 
 
+def _get_stats(recid, parent_recid):
+    """Fetch the statistics for the given record."""
+    try:
+        views = current_stats.get_query("record-view").run(recid=recid)
+        views_all = current_stats.get_query("record-view-all-versions").run(
+            parent_recid=parent_recid
+        )
+        downloads = current_stats.get_query("record-download").run(recid=recid)
+        downloads_all = current_stats.get_query("record-download-all-versions").run(
+            parent_recid=parent_recid
+        )
```

Comment:
The cached `current_stats.get_query()` is introduced in https://github.com/inveniosoftware/invenio-stats/pull/133/files
This would reduce wastefulness a bit, but require a `3.0.1` (or `3.1.0`?) release of `invenio-stats`. Then the bump would probably need to be done in my other PR (https://github.com/inveniosoftware/invenio-records-resources/pull/439) 

If you prefer, we can leave the `3.0.0` requirement and add the caching here.